Update storage-explorer-network.md

sudiptaC01 opened the pull request:
Adding Azure Front door (AFD) endpoint to the list of existing Storage Explorer update endpoints based on recent security changes based on the discussion with Craig Alvord from product group.

sudiptaC01 committed:
Update storage-explorer-network.md

Adding Azure Front door (AFD) endpoint to the list of existing Storage Explorer update endpoints based on recent security changes.

craxal reviewed (changes requested):
Please remove the link to "storageexplorerpublish". It is no longer used directly.

sudiptaC01 committed:
Update storage-explorer-network.md

Removed older link for updating storage explorer and added the azure frontdoor link.
